examples/basic_file.html

```diff
@@ -15,8 +15,8 @@
     </style>
 </head>
 <body class="markdown-body">
-    <template is="juicy-html" href="./partials/basic.html"></template>
+    <juicy-html href="./partials/basic.html"></juicy-html>
```

Comment:
This shoud be `../../examples/partials/basic.html` it's 404.

Reply:
I'll track it down, the paths should be relative to current one.

Reply:
Sorry, my bad, this can stay.